packages/manager/apps/hub/src/dashboard/dashboard.html

```diff
@@ -1,14 +1,4 @@
 <div data-ui-view>
-    <div
-        class="hub-banner"
```

Comment:
Could you please remove extra style as well?

See: https://github.com/ovh/manager/blob/master/packages/manager/apps/hub/src/dashboard/dashboard.scss#L6-L19

Reply:
👍